cmd/export_assets.go

```diff
@@ -70,6 +73,11 @@ var assetsCmd = &cobra.Command{
 		printTransformStats(len(paymentOps), numFailures)
 
 		maybeUpload(cloudCredentials, cloudStorageBucket, cloudProvider, path)
+
+		if commonArgs.WriteParquet {
+			writeParquet(transformedAssets, parquetPath, new(transform.AssetOutputParquet))
+			maybeUpload(cloudCredentials, cloudStorageBucket, cloudProvider, parquetPath)
```

Comment:
Is the just `path` path for json files?

Reply:
Sorry can you clarify? I think you might be missing a word in your question

Reply:
So `parquetPath` is the file path where the parquet files will be written. Does the existing `path` directory contain json files? Or, what's actually contained in `path` directory?

Reply:
Sorry I missed this earlier. `parquetPath` is the fully formed filename so it'll look like `bucket/scheduled_run_<date>/exported_<data type like assets>.parquet`

So this directory path should only contain .parquet files for the airflow scheduled run/batch_id.